hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestEightToNineUpgrade.scala

```diff
@@ -107,6 +111,92 @@ class TestEightToNineUpgrade extends RecordLevelIndexTestBase {
     checkResultForVersion8(payloadClass)
   }
 
+  @Test
+  def testUpgradeDowngradeMySqlDebeziumPayload(): Unit = {
```

Comment:
nit: we might want to decouple the upgrade and downgrade tests from `RecordLevelIndexTestBase`, which can be done separately.  @lokeshj1703 could you file a JIRA to track this?